src/controllers/listController.js

```diff
@@ -0,0 +1,37 @@
+const { listService } = require("../services");
+const { catchAsync } = require("../middlewares");
+
+const getSeperatedList = catchAsync(async (req, res, next) => {
+  const { 
+    query: {
+      address, 
+      category, 
+      menu,
+      limit
+    } 
+  } = req;
+
+  if (limit > 100) throw new Error("Too much Stores");
+
+  if ( !address && !category & !menu ) {
```

Comment:
```suggestion
  if ( !address && !category & !menu ) {
```
`!category & !menu`는 (bit AND)는 의도하신 건지 확인 부탁드립니다.